boards/nitropad-ns50/nitropad-ns50.config

```diff
@@ -1,4 +1,5 @@
 # Nitrokey Nitropad NS51 board configuration
+Note: for reference, othe GOP enabled FB board is librem_11
```

Comment:
Probably should put a `#` here for the comment right?  Also in the nv41 config

Reply:
@JonathonHall-Purism oups. Was neglected but will fix now thanks

Reply:
@JonathonHall-Purism fixed under https://github.com/linuxboot/heads/pull/1640/commits/1269c2cdc1535aa9f7c0a1ec42b64afc0664a314 thanks